Source/ITab_Lock.cs

```diff
@@ -141,32 +143,31 @@ protected override void FillTab()
             }
             if (Clipboard.StoredState.HasValue && Widgets.ButtonText(pasteButtonsRect, "Locks_Paste".Translate()))
             {
-                Data.WantedState.CopyFrom(Clipboard.StoredState.Value);
+                SetWantedStateData(SelDoor, Clipboard.StoredState.Value);
```

Comment:
It may cause reference issue.... in this case you need to create new object. Thats why there was `copyFrom` method call

Reply:
Going to fix this in a moment. Going to test the changes right now to make sure all is well.